### DISCUSSION TITLE
Does Keda supports Vault based trigger auth with Vault namespaces?

### DISCUSSION THREAD

**Discussion by chapurlatn** [kedacore/keda#2080] [Ideas]
Hi,

I'm starting using Keda which is, by the way, really cool and 100% fits our needs!!

We're using an enterprise Vault appliance with several Vault namespace (secret isolation).

The TriggerAuthentication spec seems to not provide any way to pass a given Vault namespace. Also, I checked Keda code and the corresponding handler do not implement namespace handling ([https://github.com/kedacore/keda/blob/main/pkg/scaling/resolver/hashicorpvault_handler.go](https://github.com/kedacore/keda/blob/main/pkg/scaling/resolver/hashicorpvault_handler.go)).
Here's an example of how to support namespace: [https://github.com/hashicorp/vault/blob/main/api/client.go#L486](https://github.com/hashicorp/vault/blob/main/api/client.go#L486)

Did I miss something? Are you open to some PR about that?

**Comment by tomkerkhove** [comment]
We are definately open for contributions! Feel free to open a feature request for this to start tracking it and we are happy to review a PR if you're up for it!

**Reply by tomkerkhove** [reply]
Tracking in https://github.com/kedacore/keda/issues/2084
